irods/test/data_obj_test.py

```diff
@@ -1860,6 +1860,41 @@ def test_access_through_resc_hierarchy__243(self):
         finally:
             s.resources.remove('parent')
 
+
+    def test_temporary_xml_mode_change_with_operation_as_proof__issue_586(self):
+        from irods.message import (XML_Parser_Type, current_XML_parser, string_for_XML_parser)
+        active_xml_for_thread = lambda : string_for_XML_parser(current_XML_parser())
```

Comment:
Consider renaming this variable. I understand what it means, but it feels like it's missing a word or two. Here's one suggestion. Feel free to use different wording if you prefer.
```
active_xml_parser_for_thread
```

Reply:
Done!